### PR TITLE
ENH: Make SegmentsTableView functions available from Python

### PR DESCRIPTION
**Pull request by cpinter**
This is needed to use the SegmentationVerification extension

### CODE REVIEW

**Comment by cpinter** [comment]
See PW project https://projectweek.na-mic.org/PW41_2024_MIT/Projects/SegmentationVerificationModuleForFinalizingMultiLabelAiSegmentations/

File: Modules/Loadable/Segmentations/Widgets/qMRMLSegmentsTableView.h [hunk 1]
```diff
@@ -76,7 +76,7 @@ class Q_SLICER_MODULE_SEGMENTATIONS_WIDGETS_EXPORT qMRMLSegmentsTableView : publ
   Q_INVOKABLE QTableView* tableWidget();
 
   /// Return number of segments (rows) in the table
-  int segmentCount() const;
+  Q_INVOKABLE int segmentCount() const;
```
Comment (lassoan) [inline Modules/Loadable/Segmentations/Widgets/qMRMLSegmentsTableView.h:79]: Shouldn't we make it a property instead?

**Review by lassoan — COMMENTED** [review]
Thanks, it looks good, just have a look at the question I added inline.

**Comment by cpinter** [comment]
I updated the commit with making `segmentCount` a property.

**Review by lassoan — APPROVED** [review]
Looks good, thank you. The CI failure is due to using an older Slicer docker image (up-to-date at the time of the PR was created) with the latest Slicer source code.

**Pull request merged**